android-build/sdk-27/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM dubit/android-build-image:base
+
+# --- Install the required build packages, tools and SDKs ---
+# * To get a list of available packages run: sdkmanager --list
+
+# --- Update the android SDK repositories ---
+RUN sdkmanager "platform-tools"
+
+# --- Install android SDK
+# * If you are installing multiple SDK packages keep these in descending order!
+RUN sdkmanager "platforms;android-27"
```

Comment:
There are a lot of layers built up from base to build image, number of layers is less of an issue for more recent docker versions but being smarter (note: not necessarily simply reducing) with layers will help with file size and build times

Reply:
Not sure what you want changed here. There are just a lot of SDKs and stuff we need to install. The base image is the one that contains most of the layers for the image. The leaf images are pretty much reduced to their bare minimum.

You could base the SDK images on each other to reduce it a little but the builder is smart enough to reuse the layers if you are building them together.